data/data.go

```diff
@@ -26,11 +26,42 @@ const EffxYamlPattern = "(.+\\.)?effx\\.ya?ml$"
 
 var effxYamlRegex = regexp.MustCompile(EffxYamlPattern)
 
+// EffxYamlPattern is the regex pattern for yaml files
+const GitPattern = ".git"
+
 // EffxYaml provides a data structure and methods for interacting with effx yamls
 type EffxYaml struct {
 	FilePath string
 }
 
+func convertToRelativePath(absoluteDir string) (string, error) {
```

Comment:
is there is an easier way to do this, please let me know haha

Reply:
Are you trying to find the relative path from the `.git` directory? Relative from the point of execution probably doesn't help much.

My guess is you want to start with the current working directory and check parent folders until you find the root repository. This approach only gets more complicated with submodules.

Another idea is to use CI variables to get repository names and URLs. You could then parse the absolute directory to create relative paths.

Reply:
> My guess is you want to start with the current working directory and check parent folders until you find the root repository. This approach only gets more complicated with submodules

that's the approach I'm using right now. I guess the advantage of this over the CI env is this can be resolved outside of a ci environment.

Reply:
This approach can be a fallback? CI env variables are guaranteed to be correct while this technique is a heuristic.